skfda/ml/regression/_linear_regression.py

```diff
@@ -50,9 +51,24 @@
 
 
 class LinearRegression(
-    BaseEstimator,  # type: ignore
+    BaseEstimator,   # type: ignore
     RegressorMixin,  # type: ignore
 ):
+
+    """
+    .. deprecated:: 0.8
+    Use covariate parameters of type pandas.FDataFrame in methods
+    fit, predict.
+
+    """
+
+    warnings.warn(
+        "Usage of arguments of type FData, ndarray or a sequence \
```

Comment:
The only deprecated case is passing a sequence. FData and ndarray are still fine.